src/tensors/abstracttensor.jl

```diff
@@ -499,3 +648,33 @@ function Base.convert(::Type{Array}, t::AbstractTensorMap)
         return A
     end
 end
+
+# Show and friends
+# ----------------
+Base.dims2string(V::HomSpace) = join(dim.(codomain(V)), '×') * "←" * join(dim.(domain(V)), '×')
+
+Base.summary(io::IO, t::AbstractTensorMap) = tensor_summary(io, t, space(t))
+
+function tensor_summary(io::IO, t, V)
+    print(io, Base.dims2string(V), " ")
+    Base.showarg(io, t, true)
+    return nothing
+end
```

Comment:
Is it useful to split this up and not just define `Base.summary` in this way? There don't seem to be any other call sites of `tensor_summary`. Is there any use case for not having `V = space(t)` ?